rollup.config.js

```diff
@@ -0,0 +1,6 @@
+const config = {
```

Comment:
I think you could just `export default {` instead of making the variable `config` - not important though.